### ISSUE TITLE
Stethascope + Webbing on a medical outfit = Derpery

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#19525]
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
e803fb3f284ce39fb87b227867a9f063413042e2

# Description

Stethescopes can clip to a paramedic or medical doctor jumpsuit for ease of access, however with webbing they are just a (suprisingly expensive) accessory
# Steps to Reproduce

Put on webbing, attempt to insert stethescope, discover this neat feature and !!FEATURE!!
# What you Expected

To put it in the webbing
# What Actually Happened

Shits fucked yo

**Comment by SonixApache** [comment]
right click uniform and 
use the remove accessory verb

**Comment by SGuivenne** [comment]
yes, but normally you can left click grab it, thats a whole extra step.

**Comment by Blithering** [comment]
If you wanted it in the webbing you could click the inventory slot  rather than the jumpsuit
